mailbox/api/src/main/java/org/apache/james/mailbox/AttachmentManager.java

```diff
@@ -36,7 +37,7 @@
 
     List<Attachment> getAttachments(List<AttachmentId> attachmentIds, MailboxSession mailboxSession) throws MailboxException;
```

Comment:
I think when working on the upload, @chibenwa saw that it could be made reactive, and contributed the change. I get your point that yes it would be better to make the rest reactive too, but it's not the goal to refactor everything here. PR is big enough :)

Reply:
agree